src/libc/include/setjmp.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _SETJMP_H_
+#define _SETJMP_H_
+
+struct __jmp_buf_tag {};
```

Comment:
I tried to enable `-Wall -pedantic-errors` and I'm getting the following:

```clang
include/setjmp.h:4:1: error: empty struct is a GNU extension [-Werror,-Wgnu-empty-struct]
```
It is indeed undefined behavior. I guess we should get rid of that but I'm not sure about how we should proceed and I'd like to discuss this with you.

Reply:
Yeah, this is because I just couldn't be bothered with the complexity of https://github.com/ensc/dietlibc/blob/master/include/setjmp.h it's also highly tied to the underlying architecture so I think it's non trivial to fully fix. Although, in the short term I could complete the struct so as to not throw compilation warnings.

Also, in the future I think it would also be best for use to move to `wasi-libc`

Reply:
cc @chambart who is a fan of setjmp and longjmp I'm sure :-)

Reply:
will fix later, see #103